poi/src/main/java/org/apache/poi/sl/usermodel/TextRun.java

```diff
@@ -79,6 +80,37 @@ enum FieldType {
     void setFontColor(PaintStyle color);
 
 
+    /**
+     * Returns the font highlight (background) color for this text run.
+     * This returns a {@link SolidPaint}, or null if no highlight is set.
```

Comment:
You could create a new interface in this package called `HightlightColorSupport` or something similar. Add these methods there instead. HSLFTextRun will not implement HightlightColorSupport. XSLFTextRun will.

DrawTextParagraph can check if the text run is an instanceof HightlightColorSupport.

Reply:
Ok sure, let me take a look and update.

Reply:
Code updated, ready for review